core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -320,6 +323,44 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
     }
   };
 
+  public static class HslaColorFormatter extends BaseFormatter {
+
+    public HslaColorFormatter() {
+      super("hslaColor", false);
+    }
+
+    @Override
+    public void apply(Context ctx, Arguments args, Variables variables) throws CodeExecuteException {
+      Variable var = variables.first();
+      JsonNode node = var.node();
+
+      int hue = node.path("hue").asInt();
+      double saturation = node.path("saturation").asDouble();
+      double lightness = node.path("lightness").asDouble();
+      double alpha = node.path("alpha").asDouble();
+      if (saturation <= 1) {
```

Comment:
Should this check for `> 0`? Also, should this print an error if the saturation/lightness values are outside the range?

Reply:
i think we need to allow for out of range values until we fix the local color override field to output decimals

Reply:
handled offline: this should confine lightness and saturation to only values between 0 and 1.  we will prioritize fixing the LCO field to output decimals